### DISCUSSION TITLE
p-multiSelect takes 20 - 30 seconds to select all options

### DISCUSSION THREAD

**Discussion by Ritik2411** [primefaces/community#1538] [PrimeNG] [answered]
p-multiSelect takes 20 - 30 seconds to select all options if number of options are 10000 or more than that. I enable virtual scroll but it stills takes long time.

**Comment by dandoteverest** [comment]
To add to that...I've been seeing single selections take 2-3 seconds, and select-all 4-5 seconds, with as many as 250 entries.

**Reply by Ritik2411** [reply]
@dandoteverest 
Have u enabled virtual scrolling? It should fix your problem.

**Reply by dandoteverest** [reply, marked as answer]
Ah, yes, I was not aware that VirtualScroller fixed that issue. Thanks!
